inventory-generation/identity-management/site.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Generate identity-management inventory based off of engagement.json
+  hosts: local
+  gather_facts: false
+  tasks:
+
+  - name: "Fail If Working Directory Is Not Provided"
+    fail:
+      msg: "directory var needs to be provided in order to ensure a safe working directory"
+    when:
+      - directory is undefined or (directory | trim) == ""
+
+  - name: "Read Engagement Data"
+    set_fact:
+      json_data: "{{ (lookup('file', directory + '/iac/engagement.json') | from_json).engagement_users | to_json }}"
```

Comment:
I believe the correct path is `directory + '/engagement.json'`. See [here](/rht-labs/lodestar-automation/blob/master/translate-engagement-data/site.yml#L8) for example.

Reply:
Fixed